zcash_keys/src/keys.rs

```diff
@@ -533,21 +546,27 @@ impl UnifiedAddressRequest {
         #[cfg(feature = "transparent-inputs")]
         let _has_p2pkh = true;
 
-        Self::new(_has_orchard, _has_sapling, _has_p2pkh)
+        Self::new(_has_orchard, _has_sapling, _has_p2pkh, None, None)
     }
 
     /// Construct a new unified address request from its constituent parts.
     ///
-    /// Panics: at least one of `has_orchard` or `has_sapling` must be `true`.
-    pub const fn unsafe_new(has_orchard: bool, has_sapling: bool, has_p2pkh: bool) -> Self {
-        if !(has_orchard || has_sapling) {
-            panic!("At least one shielded receiver must be requested.")
+    /// Panics: at least one of `has_orchard`, `has_sapling`, or `has_p2pkh` must be `true`.
```

Comment:
```suggestion
    /// Panics: at least one of `has_orchard` or `has_sapling` must be `true`.
```

Reply:
Resolved in 673e1a21d28209103dce50ede358491e5978d007